_src/css/helpcenter/_fonts.scss

```
@@ -20,6 +20,12 @@ body {
   font-family: Graphik-Regular, sans-serif;
 }
 
+// bold text doesnt stand out in graphik regular
+body strong {
```

We can skip the `body` here (because everything is part of the `body`), and just write
```suggestion
strong {
```